schemas/http.yml

```diff
@@ -26,6 +26,20 @@
 
       example: GET, POST, PUT, PoST
 
+    - name: request.mime_type
+      level: extended
+      type: keyword
+      short: Mime type of the body of the request.
+      description: >
+       Mime type of the body of the request.
+
+       This value must only be populated based on the content of the request
+       body, not on the `Content-Type` header. Comparing the mime type of a
+       request with the request's Content-Type header can be helpful in detecting
+       threats or misconfigured clients.
+
+      example: image/gif
```

Comment:
Great choice for the example value 💯

Reply:
My favorite mime type 😃 

![gif mime example](https://camo.githubusercontent.com/d573baad95ebc4d31bf476be1fddf9ee07231e39/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f3850316f4f324a62725a4b327553596e4c362f67697068792e676966)